recipes/s2s/src/aiwq_utilities.py

```diff
@@ -0,0 +1,183 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024-2025 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-FileCopyrightText: All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import os
+from datetime import datetime, timedelta
+
+import numpy as np
+import xarray as xr
+from AI_WQ_package import retrieve_evaluation_data
+
+
+def get_verif_data(
+    forecast_date: str, variable: str
+) -> tuple[xr.DataArray, xr.DataArray]:
+    """
+    Retrieve official AIWQ verification data for a given forecast date and variable.
```

Comment:
Nit can you bump these first sentences up to start on the same line as the """

For consistency, thanks!